### PR TITLE
JavaScript-I - homework

### PR DESCRIPTION
**Pull request by lightofdavinci**

### CODE REVIEW

**Comment by lightofdavinci** [comment]
I was in a hurry and afraid of using the forEach method in Object.js file because of a warning I read in comments. Here is a more elegant example that shows my understanding of how forEach works.

File: src/callbacks.js [hunk 1]
```diff
@@ -61,21 +77,31 @@ multiplyNums(5, 10, (product) => {
 // Pass true to the callback if it is, otherwise pass false
 // code here
 
+const contains = (arr,item,cb) => {
+  arr.indexOf(item) !== -1 ? cb(true) : cb(false);
```
Comment (sperrye) [inline src/callbacks.js:81]: Fancy

File: src/closure.js [hunk 2]
```diff
@@ -5,17 +5,36 @@ const counter = () => {
   // Example: const newCounter = counter();
   // newCounter(); // 1
   // newCounter(); // 2
+  let x = 0;
+  return () => {
+    return x += 1;
```
Comment (sperrye) [inline src/closure.js:10]: You can also use the `++/--` operators

File: src/objects.js [hunk 3]
```diff
@@ -5,22 +5,43 @@ const keys = (obj) => {
...
 };
 
 const values = (obj) => {
   // Return all of the values of the object's own properties.
   // Ignore functions
   // http://underscorejs.org/#values
+  const result = [];
+  const props = Object.keys(obj);
+  for (let i = 0, leng = props.length; i < leng; i++) {
+    if (typeof props[i] !== 'function') {
```
Comment (sperrye) [inline src/objects.js:18]: Why are you testing typeof for function?

**Review by sperrye — COMMENTED** [review]
Great work

**Pull request closed without merging**